client/src/components/Loading/CircularLoading.js

```diff
@@ -0,0 +1,28 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+class CircularLoading extends PureComponent {
```

Comment:
Renamed from `LoadingIndicator` to make it clear what it is (since we already have a different loading component here)